.angular-cli.json

```diff
@@ -21,6 +21,11 @@
       "styles": [
         "styles.scss"
       ],
+      "stylePreprocessorOptions": {
+        "includePaths": [
+          "./"
```

Comment:
What does this do?

Reply:
This means global `.scss` partials are detected and known at a base level, so when included in components they don't need relative paths. This resolves some issues I had getting the font assets to load across the different components.

E.g. `@import ../../'global_variables'` becomes `@import 'global_variables'`

However perhaps the`"includePaths": ["./"]` is too broad? I can instead specifically point it to files if that's a concern. I.e:

```
"includePaths": [
    "./src/_global_variables.scss"
]
```

Reply:
Nice!! That's super handy.